test/middleware/PostQuery.test.ts

```diff
@@ -28,25 +30,24 @@ describe('Test API key header verification', () => {
             complexity: 3,
             timestamp: 100,
             tokens: -1,
+            success: true,
+            loggedOn: 0,
         };
-        postQuery = new PostQuery(mockURI, mockProjectID, mockQueryData);
-        const newPost = () => postQuery.post();
+        const newPost = () => postQuery(mockURI, mockProjectID, mockQueryData);
 
         await expect(newPost).rejects.toThrow(`[gatelog] Query data cannot be negative.`);
     });
 
     test('Error thrown when resulting projectID of query does not match the one entered', async () => {
```

Comment:
This test title reads like you are expecting an error to be thrown but then you're expecting an an error not to be thrown on line 44?

Reply:
deleted this test